pydarn/plotting/fan.py

```diff
@@ -119,54 +144,58 @@ def plot_fan(cls, dmap_data: List[dict], ax=None, scan_index: int = 1,
         grndsct
             n_beams x n_gates numpy array of the scan data
             (for the selected parameter)
-        dtime
-            datetime object for the scan plotted
-
         See Also
         --------
             plot_fov
         """
-
         # Get scan numbers for each record
         beam_scan = build_scan(dmap_data)
 
+        if isinstance(scan_index, dt.datetime):
+            # loop through dmap_data records, dump a datetime
+            # list where scans start
+            scan_time = scan_index
+            scan_index = 0
+            found_match = False
+            for rec in dmap_data:
+                rec_time  = time2datetime(rec)
+                if rec['scan'] == 1:
```

Comment:
```suggestion
                if abs(rec['scan']) == 1:
```
For above issue with scan values being -1 in the new datetime to scan_index changes.